### PR TITLE
Feature/feature header

### PR DESCRIPTION
**Pull request by renanrhoden**
Finish Header for the restaurant cards screen. Couldn't  find by now a way to add custom fonts to the projet but the project has the fonts that may be added to the Header in a next feature.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-      <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
-        <Text>Changes you make will automatically reload.</Text>
-        <Text>Shake your phone to open the developer menu.</Text>
-      </View>
+      <Header 
+      title="Let's Lunch" 
+      backgroundColor="#ffa65c" 
+      titleColor="#f2f2f2"
+      />
```
Comment (Morbix) [inline App.js]: close in the same line of the last attribute

**Comment by Morbix** [comment]
@renanrhoden who said that you need a custom font? will it contribute with our goal?

File: src/components/LibraryList.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+import { connect } from 'react-redux';
+
+class LibraryList extends Component {
+  render() {
+    console.log(this.props);
```
Comment (Morbix) [inline src/components/LibraryList.js]: remove logs

File: src/components/LibraryList.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    return <Text>Teste</Text>;
```
Comment (Morbix) [inline src/components/LibraryList.js]: Avoid mock code or add some TODO mark, per example:

```
return <Text>Teste</Text>; // TODO implement props value
```

File: src/components/LibraryList.js (hunk 2 above)
Comment (renanrhoden) [inline src/components/LibraryList.js]: sorry, this should not even be there, my bad not deleting when I reused our components from the course

File: src/components/LibraryList.js (hunk 3 above)
Comment (renanrhoden) [inline src/components/LibraryList.js]: i must say that this line is from your repo but this file i forgot to delete

File: App.js (hunk 1 above)
Comment (renanrhoden) [inline App.js]: the course taught us to do like that, so did the eslint. Should I keep that way or change it?

**Pull request merged**